src/resolve/resolvectl.c

```diff
@@ -2627,14 +2712,13 @@ static int native_help(void) {
                "                                                              (default: yes)\n"
                "     --raw[=payload|packet]    Dump the answer as binary data\n"
                "     --legend=BOOL             Print headers and additional info (default: yes)\n"
-               "\nSee the %s for details.\n"
-               , program_invocation_short_name
-               , ansi_highlight()
-               , ansi_normal()
-               , ansi_highlight()
-               , ansi_normal()
-               , link
-        );
+               "\nSee the %s for details.\n",
```

Comment:
comma placement is unrelated to the change

Reply:
Yeah, I think this change should be removed. Those commas are intentionally and consistently placed like that to make it easier to edit the multiline string. If we were to decide that we don't want that and want to move the commas, this should be done for all the codebase.

Reply:
I'll see about making a PR for the entire codebase.